### PR TITLE
#155820624 Launch detail Activity from main Activity

### PR DESCRIPTION
**Pull request by malfahad**
#### What does this PR do?
- Utilizes Intents to launch the developer detail activity from the Main Activity. 

#### Description of Task to be completed?
- Add a button to the main activity
- Create and Add a new Activity to the `manifest file`
- Use click listeners and intents to navigate between activities.

#### How should this be manually tested?
- Fetch and checkout into this branch `ft-launch-detail-155820624`
- Launch the app in an emulator
- Use the launch button to navigate between activities.
 
#### What are the relevant pivotal tracker stories?
 [#155820624](https://www.pivotaltracker.com/story/show/155820624)

#### Screenshots

Screen Shot 2019-03-22 at 12 13 05 PM

Screen Shot 2019-03-22 at 12 13 19 PM

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -16,6 +16,7 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name=".DevDetailActivity"></activity>
     </application>
 
 </manifest>
```
Comment (ongebo) [inline app/src/main/AndroidManifest.xml]: Add a new line at the end of this file.

File: app/src/main/java/com/andela/javadevsnairobi/MainActivity.java [hunk 2]
```diff
@@ -1,20 +1,35 @@
...
         setContentView(R.layout.activity_main);
 
         devsRecyclerView = findViewById(R.id.devs_recycler_view);
         devsRecyclerView.setLayoutManager(new LinearLayoutManager(this));
-    }
+        launchBtn = findViewById(R.id.launch_btn);
+        launchBtn.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Intent intent = new Intent(v.getContext(),DevDetailActivity.class);
```
Comment (ongebo) [inline app/src/main/java/com/andela/javadevsnairobi/MainActivity.java]: Add a single space after the comma in your arguments list (line 29).

**Review by ongebo — COMMENTED** [review]
It's also good practice to format your files before committing. Use the keyboard shortcut `option` + `command` + `L` on Mac in Android Studio, that would have resolved the error on `line 29` in `MainActivity.java` and set the tab spacing for that file to `4 spaces`.

**Review by ongebo — APPROVED** [review]
Nice work @malfahad, looks good to me. I'm merging this in.

**Comment by malfahad** [comment]
Thanks for the heads up. @ongebo , I will use that shortcut next time

**Pull request merged**